src/app/styles/overrides.scss

```diff
@@ -302,3 +302,7 @@ common-date[label="Marriage date (optional)"] {
     padding-left: 0;
   }
 }
+
+.horizontal-line {
+  border-bottom: 1px solid black;
+}
```

Comment:
I think this class might be in additions.scss already, did you try running the app without these three lines?

Reply:
I just saw it. Will delete the one in override.scss

Reply:
Nice, everything else looks great